nextjs/src/modules/shared/components/AudioPlayer/hooks/AudioPlayer.js

```diff
@@ -13,6 +13,10 @@ export const useAudioPlayer = (audioRef, progressBarRef) => {
     progressBarRef.current.max = seconds;
   };
 
+  useEffect(() => {
+    onLoadedMetadata();
+  }, [audioRef?.current?.loadedmetadata, audioRef?.current?.readyState]);
```

Comment:
`audioRef` appears to be a required prop so it will always be there no need for the first `?`

`audioRed.current?.load.......`

I actually just learned this from James' optional chaining video